tests/test_views.py

```diff
@@ -138,7 +138,7 @@ def test_post_bucket(app, dummy_location):
 # def test_get_object_list(app, dummy_objects):
 
 
-def test_get_object_get(app, objects):
+def TODO_test_get_object_get(app, objects):
```

Comment:
Unfortunately @ddaze we did double work as I also implemented some views tests (which also take permissions into account). I have the test ``test_get_object_get`` and ``test_get_object_get_404`` temporarily. I will have to merge those views together, so the ``invenio_files_rest/test_permissions.py`` file will disappear on behalf of the views tests which will live here instead.